main.js

```diff
@@ -30,5 +30,29 @@ client.on('ready', () => {
         }
     });
 });
+client.on('message', message => { //Ten event wykonuje się, gdy bot wykryje wiadomość
+    const pingEmbed = new Discord.MessageEmbed()
+	.setColor('#eb1540')
+	.setTitle('Nie wołaj wszystkich!')
```

Comment:
Na tłumaczenia jest plik `commands.js`. A jeśli to tłumaczenie tam kontekstowo nie pasuje, to warto zrobić nowy plik i tam wrzucić tłumaczenia - powód analogiczny jak w poprzednim komentarzu.